addons/l10n_de/migrations/1.1/post-migrate_update_amls.py

```diff
@@ -0,0 +1,46 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    # The tax report line 68 has been removed as it does not appear in tax report anymore.
+    # But, it was referenced in the account.sales.report
+    # So, we update amls of this line only, to make this report consistent.
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
```

Comment:
Is this enough to have access to all the companies or should we search some more?

Reply:
`account.account.tag` does not depend on companies, so it is no issue (and the rest is directly done in sql)